src/dex-ui/pages/CreateProposal/CreateProposal.tsx

```diff
@@ -25,6 +26,14 @@ export const CreateProposal = (props: any) => {
   const handleTitleChange = (event: ChangeEvent<HTMLInputElement>) => {
     setTitle(event.target.value);
   };
+  /**
+   * Temporarily saving the values for new AddNewTextComponent.
+   * This should be replaced with Formik in the future.
+   * */
+  const [value, setValue] = useState("");
```

Comment:
As of now, just storing the HTML text from editor in future developments we need to store the complete object for text proposal including title and link to discussion along with html text